docs/guide/shop_features/catalog.md

```diff
@@ -38,8 +38,10 @@ to learn how to change the data provider.
 
 ## Product categories
 
-![product catalog](img/product_catalog_2.png)
+![Product category](img/product_catalog_2.png)
 
-A category represents a product group. [[= product_name_com =]] enables showing a product category page using different layouts (by default sub-categories only, products only or both on the entry page of a category). In the Back Office you can also configure if bestsellers of the group should be displayed.
+A category represents a product group.
+You can show a product category page using different layouts (sub-categories only, products only or both on the entry page of a category).
```

Comment:
```suggestion
You can show a product category page by using different layouts (sub-categories only, products only or both on the entry page of a category).
```
Does "on the entry page of a category" belong outside of the parenthesis? Could it be "different layouts on the entry page of a category"?